screens/Home.js

```diff
@@ -81,8 +81,26 @@ class Home extends React.Component {
   changeStatusBar(shouldBeHidden) {
     StatusBar.setHidden(shouldBeHidden);
   }
+  async requestUserPermission() {
+    const authStatus = await messaging().requestPermission();
+    const enabled =
+      authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+      authStatus === messaging.AuthorizationStatus.PROVISIONAL;
+  
+    if (enabled) {
+      console.log('Authorization status:', authStatus);
+    }
+  }
```

Comment:
Do we need this `console.log` in production too? Can we remove this one as well?

Reply:
Yes, Fixed